crates/rolldown/tests/esbuild/default/import_missing_common_js/artifacts.snap

```diff
@@ -18,9 +18,8 @@ var require_foo = __commonJSMin((exports, module) => {
 //#endregion
 //#region entry.js
 var import_foo = __toESM(require_foo());
-if (false) {
-	console.log((0,import_foo.default)(import_foo.x, import_foo.y));
-}
+if (false) console.log((0,import_foo.default)(import_foo.x, import_foo.y));
```

Comment:
Constant folding is not supported in oxc compersor. Oxc so far only support folding in Ternary operation.